toolbox/ui.py

```diff
@@ -136,7 +136,28 @@ def draw_umap_projections(self, utterances: Set[Utterance]):
         self.umap_ax.set_xticks([])
         self.umap_ax.set_yticks([])
         self.umap_ax.figure.canvas.draw()
-        
+
+    def setup_audio_devices(self,samplerate):
+        supported_devices=[]
+        for device in sd.query_devices():
+            try:
+                sd.check_output_settings(device=device['name'], samplerate=samplerate)
+                supported_devices.append(device['name'])
+            except Exception as e:
+                print("Unsuported device: ",samplerate,"Device: ", device['name'], "\n", e)
```

Comment:
Suggestions:
1. Fix spelling of "Unsuported"
2. Make this a warning instead of a print statement (using the warnings package)

Reply:
Do we need to explain to the user that the number being printed is the sample rate? The output currently looks like this:
```
Unsuported device:  16000 Device:  Built-in Microphone 
 No output device matching 'Built-in Microphone'
Unsuported device:  16000 Device:  Multi-Output Device 
 No output device matching 'Multi-Output Device'
```